src/chat/chatAgentRequestHandler.ts

```diff
@@ -364,7 +368,8 @@ export const createSqlAgentRequestHandler = (
         copilotService: CopilotService,
     ): Promise<GetNextMessageResponse> {
         if (sqlTools.length === 0) {
-            throw new Error("No tools to process.");
+            console.error("No tools to process.");
+            throw new Error(loc.noToolsToProcess);
```

Comment:
Is this an error that would happen during normal execution, or do we need a `sendErrorEvent()` here?